homeassistant/components/music_assistant/media_browser.py

```diff
@@ -314,6 +327,54 @@ async def build_tracks_listing(mass: MusicAssistantClient) -> BrowseMedia:
     )
 
 
+async def build_podcasts_listing(mass: MusicAssistantClient) -> BrowseMedia:
+    """Build Podcasts browse listing."""
+    media_class = LIBRARY_MEDIA_CLASS_MAP[LIBRARY_PODCASTS]
+    return BrowseMedia(
+        media_class=MediaClass.DIRECTORY,
+        media_content_id=LIBRARY_PODCASTS,
+        media_content_type=MediaType.PODCAST,
+        title=LIBRARY_TITLE_MAP[LIBRARY_PODCASTS],
+        can_play=False,
+        can_expand=True,
+        children_media_class=media_class,
+        children=sorted(
+            [
+                build_item(mass, podcast, can_expand=False)
+                # we only grab the first page here because the
+                # HA media browser does not support paging
+                for podcast in await mass.music.get_library_podcasts(limit=500)
+                if podcast.available
+            ],
+            key=lambda x: x.title,
+        ),
+    )
+
+
+async def build_audiobooks_listing(mass: MusicAssistantClient) -> BrowseMedia:
+    """Build Audiobooks browse listing."""
+    media_class = LIBRARY_MEDIA_CLASS_MAP[LIBRARY_AUDIOBOOKS]
+    return BrowseMedia(
+        media_class=MediaClass.DIRECTORY,
+        media_content_id=LIBRARY_AUDIOBOOKS,
+        media_content_type=DOMAIN,
+        title=LIBRARY_TITLE_MAP[LIBRARY_AUDIOBOOKS],
+        can_play=False,
+        can_expand=True,
+        children_media_class=media_class,
+        children=sorted(
+            [
+                build_item(mass, audiobook, can_expand=False)
+                # we only grab the first page here because the
+                # HA media browser does not support paging
+                for audiobook in await mass.music.get_library_audiobooks(limit=500)
+                if audiobook.available
+            ],
+            key=lambda x: x.title,
```

Comment:
Let's not do this. Call the "get_library_audiobooks" with the correct sort option so that will use backend supported sorting